lib/std/math.zig

```diff
@@ -397,12 +415,34 @@ test {
     _ = gamma;
     _ = lgamma;
 
-    _ = complex;
     _ = Complex;
 
     _ = big;
 }
 
+test "Complex inputs" {
```

Comment:
It'd be nice if this tested anything like the functions returning the correct type or a sensible value.